sum.py

```diff
@@ -13,6 +13,7 @@ def my_sum(a, b):
     """
     assert (is_number(a) and is_number(b)), 'Both addends should be numbers'
     return a - b
```

Comment:
return a+b

Reply:
The change should be proposed in the code, not as the comment.